app/src/androidTest/java/com/pse2022/timefy/ui/profile/ProfileUiTest.kt

```diff
@@ -0,0 +1,312 @@
+package com.pse2022.timefy.ui.profile
+
+import androidx.compose.ui.test.*
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.lifecycle.viewmodel.compose.viewModel
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.firebase.auth.GetTokenResult
+import com.google.gson.GsonBuilder
+import com.pse2022.timefy.data.AuthManager
+import com.pse2022.timefy.data.api.EventService
+import com.pse2022.timefy.data.api.UserService
+import com.pse2022.timefy.data.event.Event
+import com.pse2022.timefy.data.event.EventCategory
+import com.pse2022.timefy.data.event.EventTimeSlot
+import com.pse2022.timefy.data.event.EventVisibility
+import com.pse2022.timefy.data.user.User
+import com.pse2022.timefy.data.user.UserRelation
+import com.pse2022.timefy.ui.profile.viewModel.ProfileViewModel
+import com.pse2022.timefy.ui.theme.TimefyTheme
+import com.pse2022.timefy.ui.utils.TestTags
+import kotlinx.coroutines.flow.StateFlow
+import okhttp3.OkHttpClient
+import okhttp3.mockwebserver.MockResponse
+import okhttp3.mockwebserver.MockWebServer
+import org.junit.*
+import org.junit.runner.RunWith
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+class MockAuthManager: AuthManager {
+    override var userService: UserService
+        get() = TODO("Not yet implemented")
+        set(value) {}
+    override val isAuthorized: StateFlow<Boolean>
+        get() = TODO("Not yet implemented")
+
+    override fun authorizeUser(onSuccess: () -> Unit, onError: () -> Unit) {
+        TODO("Not yet implemented")
+    }
+
+    override fun getUserToken(): GetTokenResult? {
+        TODO("Not yet implemented")
+    }
+
+    override fun getFullName(): String {
+        return "Max Mustermann"
+    }
+
+    override fun getFirstName(): String {
+        return "Max"
+    }
+
```

Comment:
```suggestion

    override fun getUserId(): String {
        TODO("Not yet implemented")
    }
```
A method is missing here  (was added by my last pull request ) ^^

Reply:
Fixed ^^